lambdaclass/champions_of_mirra
# Add main theme to Overworld scene

Closes #54 

## Motivation

Add music to the game, and a button to mute/unmute it.

## Summary of changes

- Add main theme to the game. 
  - The music starts playing when the application starts, and never stops.
  - It's a looped song, which you'll hear in every scene. 
- Add a toggle button in the Overworld and Campaign scenes. 
  - The button **pauses** the music.
  - The button is placed in the header prefab, that's the reason for having it only in these two scenes. As this header will soon be in the rest of the scenes, the button will be added to the rest of the scenes at that time. If the Battle scene does not finally have the header, we can easily add the button in another PR.


## How has this been tested?

- Start the application with the speakers on! Notice the background music in the first scene.
- Test the audio toggle button, muting and unmuting the music.
- Make sure that the song keeps playing through out the game, without restarting after moving from the Battle scene back to the Overworld scene. 

## Checklist
